.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+jobs:
+    ci:
+        name: Build + Test
+        runs-on: macOS-latest
+        steps:
+            - uses: actions/checkout@v1
+            - uses: actions/setup-java@v1
+              with:
+                  java-version: 1.8
+
+            # Ensure the cache is clean before restoring it.
+            # This helps us stay within Github's cache size limits.
+            - name: Clean Cache
+              run: rm -rf ~/.gradle/caches
+
+            # Restore the cache.
+            # We intentionally don't set 'restore-keys' so the cache is always exactly the size our current dependencies.
+            - uses: actions/cache@v1
+              with:
+                  path: ~/.gradle/caches
+                  key: gradle-${{ runner.os }}-${{ hashFiles('**/build.gradle.kts') }}-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/buildSrc/src/main/kotlin/Library.kt') }}
+
+            - name: Ktlint
+              run: ./gradlew ktlintCheck
+
+            - name: Unit Tests
+              run: ./gradlew testDebugUnitTest
+
+            - name: Instrumentation Tests
+              uses: reactivecircus/android-emulator-runner@v2
+              with:
+                  api-level: 25
+                  arch: x86_64
```

Comment:
FYI Google [recommends only using x86_64 images for API 26+](https://www.reddit.com/r/androiddev/comments/e5nulf/emulator_29211_and_amd_hypervisor_12_to_stable/f9l6aos?utm_source=share&utm_medium=web2x) 😃

Reply:
Ah good catch! Also thanks for the great Github Action. Will update.